### PR TITLE
Class method to animate

### PR DESCRIPTION
**Pull request by rodrigosoldi**
Animate just using KDLoadingView.animate() in topView.
And just KDLoadingView.stop() to stop the animation

### CODE REVIEW

File: KDLoadingView/KDLoadingBlurView.swift [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+//  KDLoadingBlurView.swift
+//  KDLoadingView
+//
+//  Created by Rodrigo Soldi Lopes on 02/03/17.
+//  Copyright © 2017 Kaique Damato. All rights reserved.
+//
+
+import UIKit
+
+class KDLoadingBlurView: UIVisualEffectView {
```
Comment (KaiqueDamato) [inline KDLoadingView/KDLoadingBlurView.swift]: Make class public.

File: KDLoadingView/KDLoadingExtension.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+extension KDLoadingView {
+    
+    public class func animate(lineWidth: CGFloat = 2.0, size: CGFloat = 25, duration: CGFloat = 3.0, firstColor: UIColor? = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1), secondColor: UIColor? = nil, thirdColor: UIColor? = nil) {
+        guard let window = UIApplication.shared.keyWindow,
+            let topView = window.rootViewController?.view else {
+                return
+        }
+        
+        // Blur View
```
Comment (KaiqueDamato) [inline KDLoadingView/KDLoadingExtension.swift]: Delete the comment

File: KDLoadingView/KDLoadingExtension.swift [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+//  KDLoadingView
+//
+//  Created by Rodrigo Soldi Lopes on 02/03/17.
+//  Copyright © 2017 Kaique Damato. All rights reserved.
+//
+
+extension KDLoadingView {
+    
+    public class func animate(lineWidth: CGFloat = 2.0, size: CGFloat = 25, duration: CGFloat = 3.0, firstColor: UIColor? = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1), secondColor: UIColor? = nil, thirdColor: UIColor? = nil) {
+        guard let window = UIApplication.shared.keyWindow,
```
Comment (KaiqueDamato) [inline KDLoadingView/KDLoadingExtension.swift]: Maybe it's better to split the guard let and put some log explaining the problem in the else statement.

File: KDLoadingView/KDLoadingExtension.swift [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        
+        blurView.addSubview(loadingView)
+        loadingView.startAnimating()
+        blurView.loadingView = loadingView
+        
+        addSubviewWithTransitionAnimation(fromView: topView, toView: blurView)
+    }
+    
+    public class func stop() {
+        if let window = UIApplication.shared.keyWindow {
```
Comment (KaiqueDamato) [inline KDLoadingView/KDLoadingExtension.swift]: Maybe it's better to replace if let for guard let and put some log explaining the problem in else statement

File: KDLoadingView/KDLoadingExtension.swift [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+            guard let topView = window.rootViewController?.view else {
+                return
```
Comment (KaiqueDamato) [inline KDLoadingView/KDLoadingExtension.swift]: Put some log explaining the problem in else statement

File: KDLoadingView/KDLoadingExtension.swift [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+            }
+            
+            for view in topView.subviews {
+                if view.isKind(of: KDLoadingBlurView.self) {
```
Comment (KaiqueDamato) [inline KDLoadingView/KDLoadingExtension.swift]: Try to replace the if for an if let with the cast

File: KDLoadingView/KDLoadingView.swift [hunk 7]
```diff
@@ -179,3 +179,4 @@ import UIKit
         }
     }
 }
+
```
Comment (KaiqueDamato) [inline KDLoadingView/KDLoadingView.swift]: Remove the space

**Review by KaiqueDamato — CHANGES_REQUESTED** [review]
Please, update the example project with an example of what have been made.

**Review by rodrigosoldi — COMMENTED** [review]
All the requests was fixed.

**Pull request merged**